src/components/Facility/FacilityUsers.tsx

```diff
@@ -85,13 +84,22 @@ export default function FacilityUsers(props: { facilityId: string }) {
     >
       <hr className="mt-4 border-gray-200" />
       <div className="flex flex-col sm:flex-row sm:items-center sm:justify-between gap-4 m-5 ml-0">
-        <Input
-          id="search-by-username"
-          name="username"
-          onChange={(e) => updateQuery({ username: e.target.value })}
-          value={qParams.username}
-          placeholder={t("search_by_username")}
-          className="w-full max-w-sm"
+        <SearchInput
+          options={[
+            {
+              key: "username",
+              type: "text" as const,
+              placeholder: t("search_by_username"),
+              value: qParams.username || "",
+            },
+          ]}
+          initialOptionIndex={0}
```

Comment:
shouldn't this logic be applicable everywhere? 🤔 

if so, do we even need a prop? 🤔 can't we always use this logic defined inside the search input itself? 🤔 

<img width="586" alt="image" src="https://github.com/user-attachments/assets/495d31d7-1e5d-4dc2-912f-6b01744d4c04" />